newscoop/application/controllers/RegisterController.php

```diff
@@ -107,8 +107,6 @@ public function confirmAction()
                 if (!empty($values['image'])) {
                     $imageInfo = array_pop($form->image->getFileInfo());
                     $values['image'] = $this->_helper->service('image')->save($imageInfo);
-                } else {
-                    $values['image'] = $this->getUserImageFilename($user);
```

Comment:
theme change? if `$values['image']` is set then that means user wants to upload his user profile image, if its empty, then it should not save anything in database and just grab default image from theme directory